pyproject.toml

```diff
@@ -146,7 +146,23 @@ ignore_missing_imports = true
   # arguments, which don't work in 3.7, which causes mypy to bail out early if you have
   # numpy installed.
   [[tool.mypy.overrides]]
-  module = 'numpy'
+  module = [
+      'IPython',
+      'altgraph',
+      'attr',
+      'boto3',
+      'botocore',
+      'distro',
+      'jinja2',
+      'jsonschema',
+      'macholib',
+      'markupsafe',
+      'numpy',
+      'pyristent',
+      'pytest',
+      'ruamel',
```

Comment:
On this branch I'm seeing:
```console
> spack style -t mypy                                                                                                                                                                                           0.2s
==> Running style checks on spack
  selected: mypy
==> Modified files
==> Running mypy checks
lib/spack/spack/util/spack_yaml.py:26: error: Module "ruamel.yaml" has no attribute "comments"  [attr-defined]
lib/spack/spack/util/spack_yaml.py:26: error: Module "ruamel.yaml" has no attribute "constructor"  [attr-defined]
lib/spack/spack/util/spack_yaml.py:26: error: Module "ruamel.yaml" has no attribute "emitter"  [attr-defined]
lib/spack/spack/util/spack_yaml.py:26: error: Module "ruamel.yaml" has no attribute "error"  [attr-defined]
lib/spack/spack/util/spack_yaml.py:26: error: Module "ruamel.yaml" has no attribute "representer"  [attr-defined]
Found 5 errors in 1 file (checked 578 source files)
  mypy found errors
==> Error: spack style found errors
```
Not sure why I'm the only one seeing this. It's definitely using the vendored copy too, I don't have ruamel installed.

Reply:
I've seen this as well -- what files are modified vs. develop for you right now?

Reply:
None, I'm on your branch with no modified files.

Reply:
what commit is develop at

Reply:
297891152077a018acb7dd2b656e9018e46a5d5c